pb_plugins/dcsdkgen/autogen.py

```diff
@@ -42,7 +43,8 @@ def generate_reactive(request):
                                              responses,
                                              template_env)
 
-            out_file = File(plugin_name,
+            out_file = File(proto_file.package,
```

Comment:
We should fix on one parameter sequence, With `Struct.collect_structs`, `plugin_name`goes in front of `proto_file.package` and here it's swapped around